### ISSUE TITLE
Unable to import aws_ram_resource_share_accepter

### ISSUE THREAD

**Issue by cacack** [terraform-providers/terraform-provider-aws#10186]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version
```
Terraform v0.12.8
+ provider.aws v2.28.1
```

### Affected Resource(s)

* aws_ram_resource_share_accepter

### Expected Behavior

Successfully imported the resource

### Actual Behavior

```
module.project_satellite_vpc.aws_ram_resource_share_accepter.transitgw: Importing from ID "arn:aws:ram:us-east-2:123456789012:resource-share/44b605ed-97da-6010-130a-9efb4006e4a1"...
module.project_satellite_vpc.aws_ram_resource_share_accepter.transitgw: Import prepared!
  Prepared aws_ram_resource_share_accepter for import
module.project_satellite_vpc.aws_ram_resource_share_accepter.transitgw: Refreshing state... [id=arn:aws:ram:us-east-2:123456789012:resource-share/44b605ed-97da-6010-130a-9efb4006e4a1]

Error: Cannot import non-existent remote object

While attempting to import an existing object to
aws_ram_resource_share_accepter.transitgw, the provider detected that no
object exists with the given id. Only pre-existing objects can be imported;
check that the id is correct and that it is associated with the provider's
configured region or endpoint, or use "terraform apply" to create a new remote
object for this resource.
```

### Steps to Reproduce

1. `terraform import -provider=aws.transitgw_account module.project_satellite_vpc.aws_ram_resource_share_accepter.transitgw 'arn:aws:ram:us-east-2:123456789012:resource-share/44b605ed-97da-6010-130a-9efb4006e4a1'`

### Important Factoids

The named provider has an assume_role configured with full admin policy in the given account.

### References
* #10064

**Comment by Geartrixy** [comment]
I noticed the ID actually contains "rs" so I amended the ARN, for example to "resource-share/ **rs-** 44b605ed-97da-6010-130a-9efb4006e4a1", but the import ends up hanging when refreshing the state 🙃

**Comment by benoit74** [comment]
I reproduced this issue.
This should be solved at the same time than #10064, both issues are linked to the same root cause (the RAM invitation disappear after one week once accepted, cf. my comments on the other issue) that could be solved by the same code modification (but still two test cases have to be checked, the "re-plan" of the accepter after more than one week, and the import).

**Comment by bflad** [comment]
The fix for this has been merged and will release with version 2.50.0 of the Terraform AWS Provider, Thursday this week. Thanks to @benoit74 for the implementation. 👍
